clifford/_mvarray.py

```diff
@@ -29,7 +55,8 @@ def value(self):
         """
         Return an np array of the values of multivectors
         """
-        return np.array([mv.value for mv in self])
+        v_value_get = np.vectorize(lambda x: x.value, otypes=[np.float], signature='()->(n)')
```

Comment:
`np.float` raises a DeprecationWarning in numpy master. Let's fix this to use `self.layout_dtype`.